python/cog/server/http.py

```diff
@@ -278,11 +278,6 @@ async def predict(
         """
         Run a single prediction on the model
         """
-        if runner.is_busy():
-            return JSONResponse(
-                {"detail": "Already running a prediction"}, status_code=409
-            )
```

Comment:
This has been removed because we think it's not necessary -- we already catch the `RunnerBusyError` when attempting to call `predict` inside the `_predict` function.